std/conv.d

```diff
@@ -3675,11 +3675,25 @@ Target parseElement(Target, Source)(ref Source s)
  * Convenience functions for converting any number and types of
```

Comment:
Can you change "any number and types of arguments" to "one or more arguments of any type"?